pkg/master/master.go

```diff
@@ -197,12 +198,60 @@ func (m *Master) InstallAPIs(c *Config) {
 
 	// allGroups records all supported groups at /apis
 	allGroups := []unversioned.APIGroup{}
+
+	// Install autoscaling unless disabled.
+	var autoscalerStorage, autoscalerStatusStorage rest.Storage
+	if !m.ApiGroupVersionOverrides["autoscaling/v1"].Disable {
+		autoscalerStorage, autoscalerStatusStorage = horizontalpodautoscaleretcd.NewREST(
```

Comment:
It would be nice to put this in a separate function, I think the other groups all do that.